packages/core/src/routes/logto-config.ts

```diff
@@ -304,14 +304,14 @@ export default function logtoConfigRoutes<T extends AuthedRouter>(
        */
       body: z.discriminatedUnion('tokenType', [
         z.object({
-          tokenType: z.literal(LogtoJwtTokenKey.AccessToken),
+          tokenType: z.literal(LogtoJwtTokenPath.AccessToken),
```

Comment:
rename the field name as `tokenPathType` to align with the name of the enum variable?

Reply:
fieldName should be named to match current context usage,  tokenPath is not a valid namespace here?

Reply:
OK